packages/story-editor/src/components/videoTrim/useVideoTrimMode.js

```diff
@@ -109,9 +115,19 @@ function useVideoTrimMode() {
     if (selectedElement?.type !== 'video' || !selectedElement?.resource) {
       return false;
     }
-    const { local, isExternal } = selectedElement.resource || {};
-    return isVideoTrimEnabled && isTranscodingEnabled && !isExternal && !local;
-  }, [selectedElement, isVideoTrimEnabled, isTranscodingEnabled]);
+    const { id, isExternal } = selectedElement.resource || {};
```

Comment:
```suggestion
    const { id, isExternal } = selectedElement.resource;
```

There's already a `!selectedElement?.resource` check above, so at this point here we know that it exists

Reply:
There was a unit tests failure because of this line. Which is why it failed. Nothing wrong with being careful.

Reply:
I tend to disagree, one could be overly careful.

Can you point me to the unit test?

Reply:
Seems I cant replicate.